lib/seeds/course_seeder.rb

```diff
@@ -26,15 +28,19 @@ def add_section(&)
       end
     end
 
+    # rubocop:disable Metrics/AbcSize
     def course
```

Comment:
Only over the score by 2, a bit annoying but I don't think breaking it up improves it much. Just a bit of a data clump :shrug: